docs/v2/adding-new-codegenerated-resource.md

```diff
@@ -61,7 +61,29 @@ Once you have a working development environment, run the `task` command to run t
 
 ## Fix any errors raised by the code generator
 
-TODO: expand on common errors
+### \<Resource\> looks like a resource reference but was not labelled as one
+Example:
+>  Replace cross-resource references with genruntime.ResourceReference: ["github.com/Azure/azure-service-operator/hack/generated/_apis/microsoft.containerservice/v1alpha1api20210501/PrivateLinkResource.Id" looks like a resource reference but was not labelled as one. It might need to be manually added to `newKnownReferencesMap`,
```

Comment:
Suggest adding some line breaks so the message can be read without scrolling.
```suggestion
>  Replace cross-resource references with genruntime.ResourceReference: 
> ["github.com/Azure/azure-service-operator/hack/generated/_apis/microsoft.containerservice/v1alpha1api20210501/PrivateLinkResource.Id" looks like a resource reference but was not labelled as one. 
> It might need to be manually added to `newKnownReferencesMap`,
```

Reply:
Done